eq-author/src/App/questionPage/Preview/index.js

```diff
@@ -137,6 +133,12 @@ export const UnwrappedPreviewPageRoute = ({ loading, data }) => {
           </Details>
         )}
 
+        {guidance && (
```

Comment:
Was this in the wrong place?

Reply:
Yes. Wrong place. It now matches `eq-survey-runner` layout.